ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -0,0 +1,180 @@
+          return !metacardDefinitions.isHiddenTypeExceptThumbnail(type.id)
+        }).filter(function (type) {
+          return !excludedList.hasOwnProperty(type.id)
+        }).map(function (metacardType) {
+          return {
+            label: metacardType.alias || metacardType.id,
+            value: metacardType.id
+          }
+        }),
+        values: this.model.get('descriptors'),
+        value: [currentValue],
+        id: 'Attributes'
+      })
+    }))
+  },
+  setupTitleInput: function () {
+    let currentValue = this.model.get('resultTitle') ? this.model.get('resultTitle') : ''
+    this.basicTitle.show(new PropertyView({
+      model: new Property({
+        value: [currentValue],
+        id: 'Title',
+        placeholder: 'Result Form Title'
+      })
+    }))
+  },
+  setupDescription: function () {
+    let currentValue = this.model.get('description') ? this.model.get('description') : ''
+    this.basicDescription.show(new PropertyView({
+      model: new Property({
+        value: [currentValue],
+        id: 'Description',
+        placeholder: 'Result Form Description'
+      })
+    }))
+  },
+  turnOnLimitedWidth: function () {
+    this.regionManager.forEach(function (region) {
+      if (region.currentView && region.currentView.turnOnLimitedWidth) {
+        region.currentView.turnOnLimitedWidth()
+      }
+    })
+  },
+  edit: function () {
+    this.$el.addClass('is-editing')
+    this.regionManager.forEach(function (region) {
+      if (region.currentView && region.currentView.turnOnEditing) {
+        region.currentView.turnOnEditing()
+      }
+    })
+  },
+  cancel: function () {
+    this.cleanup()
+  },
+  save: function () {
+    let view = this
+    Loading.beginLoading(view)
+    let descriptors = this.basicAttributeSpecific.currentView.model.get('value')
+    let title = this.basicTitle.currentView.model.getValue()[0]
+    if(title === '')
+    {
+      let $validationElement = this.basicTitle.currentView.$el.find('> .property-label .property-validation')
+      $validationElement.removeClass('is-hidden').removeClass('is-warning').addClass('is-error')
+      $validationElement.attr('title', "Name field cannot be blank")
+      Loading.endLoading(view)
+      return 
+    }
+    let description = this.basicDescription.currentView.model.getValue()[0]
+    let id = this.model.get('formId')
+    let templatePerms = {
+      'descriptors': descriptors.flatten(),
+      'title': title,
+      'description': description,
+      'id' : id
+    }
+    this.updateResults(templatePerms)
+  },
+  updateResults: function (templatePerms) {
+    let resultEndpoint = `/search/catalog/internal/forms/result`
+    $.ajax({
+      url: resultEndpoint,
+      contentType: 'application/json; charset=utf-8',
+      dataType: 'json',
+      type: 'PUT',
+      data: JSON.stringify(templatePerms),
+      context: this,
+      success: function (data) {
+        this.message('Success!', 'Saved Result Form', 'success')
```

Comment:
✏️ Search forms currently only give an announcement if an error happens and "silently succeed" otherwise. Might be nice if the behavior was consistent between the two.

Intrigue as a whole seems to lean more towards silently succeeding (or at least not announcing "I did the thing!" for everything it does), so I think that's what I would lean towards.